modules/govuk_jenkins/templates/jobs/run_whitehall_data_migrations.yaml.erb

```diff
@@ -3,10 +3,10 @@
     name: Run_Whitehall_Data_Migrations
     display-name: Run_Whitehall_Data_Migrations
     project-type: freestyle
-    description: "This job SSHes to whitehall-backend-2.backend.<%= @app_domain -%> and runs 'cd /var/apps/whitehall ; RAILS_ENV=production       bundle exec rake db:data:migrate'"
+    description: "This job SSHes onto a whitehall-backend node and runs 'cd /var/apps/whitehall ; RAILS_ENV=production bundle exec rake db:data:migrate'"
     builders:
         - shell: |
-            ssh deploy@whitehall-backend-2.backend.<%= @app_domain -%> 'cd /var/apps/whitehall ; govuk_setenv whitehall bundle exec rake db:data:migrate'
+            ssh deploy@$(govuk_node_list -c whitehall-backend --single-node) "cd /var/apps/whitehall ; govuk_setenv whitehall bundle exec rake db:data:migrate"
```

Comment:
oh - actually I think you need `whitehall_backend`

Reply:
Good spot... Thanks @bevanloon. Fixed in 76d0c7b